src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/card/card.component.html

```diff
@@ -1,10 +1,8 @@
 <div class="card shadow-sm">
-  <div class="card-body">
-    <h4 class="card-title m-2">
-        {{ title }}
-    </h4>
-    <div class="card-text">
-      <ng-content></ng-content>
-    </div>
+  <h4 class="card-title mt-4 ml-4 mb-0">
+    {{ title }}
+  </h4>
+  <div class="card-body pl-0 pr-0">
+    <ng-content></ng-content>
```

Comment:
What is this change for?

Reply:
It is to properly align the section divider with the whole content of the card.

If you check the below to screenshots, you can see the difference between the dividers. It was much easier this way.

![Screenshot from 2022-09-29 11-07-12](https://user-images.githubusercontent.com/71764184/193251603-a281b560-fb69-476f-ba2

vs

![Screenshot from 2022-09-23 15-44-25](https://user-images.githubusercontent.com/71764184/193251619-c0e48e0d-35a7-4a0d-bd17-caaabd055b8c.png)
a-45593535e27f.png)